lib/identity_job_log_subscriber.rb

```diff
@@ -37,7 +37,7 @@ def enqueue_at(event)
 
     json = default_attributes(event, job)
 
-    if ex
+    if ex && should_not_warn?(job, ex)
       json[:exception_class] = ex.class.name
       json[:exception_message] = ex.message
```

Comment:
I think one of the goals here is that we still record the exception class and exception message. In #5871 we added `exception_class_warn` so we could still persist the data in our logs, WDYT about adding that to all the rest of these methods too?

Reply:
yep, I like both the pattern implemented here and the `_warn` pattern, would be great to combine them.

Reply:
Updated.